pkgs/development/python-modules/kivy/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, buildPythonPackage, fetchPypi, fetchpatch
+, pkg-config, cython
+, kivy-garden, requests, docutils, pygments
+, mesa, mtdev, SDL2, SDL2_image, SDL2_ttf, SDL2_mixer, gst_all_1
+, pillow
+}:
+
+buildPythonPackage rec {
+  pname = "Kivy";
+  version = "2.0.0rc4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1l19kfsprgrh2a19s11zbwgjdgbwh4lvnwg27rri01kgvsisbhqn";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    cython
+  ];
+
+  buildInputs = [
+    kivy-garden
+    requests
+    docutils
+    pygments
```

Comment:
requests and pygments should probably go into `propagatedBuildInputs`  as those are python modules

docutils should go in NativeBuildInputs if it's not being used at runtime.

Reply:
Fixed as well. I left it like this because there was some previous discussion as to whether `pygments` are actually a runtime dependency for kivy. I think it is though.